cypress/support/cypress-commands/commands.js

```diff
@@ -539,6 +539,15 @@ Cypress.Commands.add('setResponse', (beforeOperation, scenarioName) => {
     cy.sendMessagetoPlatforms(requestMap).then((result) => {
       fireLog.isTrue(result.success, 'Response for marker creation: ' + JSON.stringify(result));
     });
+  } else if (
+    beforeOperation.hasOwnProperty(CONSTANTS.INTERACTIONS_METRICS) &&
+    beforeOperation.interactionsMetrics === true
+  ) {
+    cy.startOrStopInteractionsService(CONSTANTS.INITIATED).then((response) => {
+      if (response) {
+        Cypress.env(CONSTANTS.IS_INTERACTIONS_SERVICE_ENABLED, true);
+      }
+    });
```

Comment:
lets make sure that any error which occurs here is handled properly and not cause the rest of the tests to fail/skipped

Reply:
Sure

Reply:
We should do PoC on printing the before operation logs. Silently running the rest of execution might cause complex debug problems.